pages/includes/login_functions.inc.php

```diff
@@ -65,14 +65,14 @@ function check_login($dbc, $email = '', $pass = '') {
 * $loc is the url that the user is redirected to
 * $tim is the time in milliseconds the message is displayed for
 */
-function redirect_js($loc, $tim) {
+function redirect_js($loc, $tim = 1000) {
 	echo "
 	<script>	
 		setTimeout(
 			function() {
 				console.log(\"Redirection timer on\");";
 	echo "window.location.href = '" . $loc . "';";
 	echo "	}
-		, 1000);
+		, $tim);
```

Comment:
I'm not sure why this is happening now. Please review the entire code throughout the website to see where else this was called.

Reply:
Hi, I did a global search and here’s what I found:

- there were only two files where the parameter was set to a number other than 1000 (including this one above)
- All times that the function was called, it gives the $tim argument a value. 

I guess the development team just didn’t notice the times were all the same?

Reply:
You're right. Thanks for the response and info. I am merging the commits now.